static/scss/partials/glocal-menu.scss

```diff
@@ -20,6 +20,14 @@ glocal-menu {
   right: 28px;
   border-radius: 8px;
   box-shadow: 0 7px 12px -3px rgba(28, 28, 29, .502);
+  width: auto;
+  max-width: $screen-sm;
+
+    @media screen and (max-width: 520px) {
```

Comment:
Magic breakpoint of 520px here, which is roughly `$screen-sm` + some leg room to account for left/right padding.